src/components/LmsConfigurations/common.jsx

```diff
@@ -0,0 +1,21 @@
+import NewRelicService from '../../data/services/NewRelicService';
+
+export const handleErrors = (error) => {
+  const errorMsg = error.message || error.response?.status === 500
+    ? error.message : JSON.stringify(error.response.data);
+  NewRelicService.logAPIErrorResponse(errorMsg);
+  return errorMsg;
+};
+
+/**
+   * Validates this form. If the form is invalid, it will return the fields
+   * that were invalid. Otherwise, it will return an empty object.
+   * @param {FormData} formData
+   * @param {[String]} requiredFields
+   */
+export const validateLmsConfigForm = (formData, requiredFields) => {
```

Comment:
I plan on updating the other LMS configurations to use these, but want to get my own component fully functioning first.